src/client.py

```diff
@@ -18,23 +20,42 @@
 import base64
 import pickle
 
+INCONN_THRESH = 128
+OUTCONN_THRESH = 8
 BUFF_SIZE = 2048
 
-
 class Client(Node):
```

Comment:
Looks like this got completely botched during the merge. There were many changes made in `Client`, and I'm not sure why there were so many additions. Most of the additions you made will cause issues.